Snackbar/snackbar.html

```diff
@@ -0,0 +1,92 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Snackbar</title>
+
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+
+    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="/Snackbar/snackbar.css">
+</head>
+
+<body>
+    <header class="home-header flex">
+        <img class="logo" src="/images/comp-lib-logo.png" alt="Brand-logo">
+        <h1 class="brand">Garnish UI</h1>
+        <ul class="nav-links flex">
+            <li><a class="nav-link" href="/index.html">Home</a></li>
+            <li><a class="nav-link link-active" href="/documentation.html">Documentation</a></li>
+        </ul>
+    </header>
+
+    <main class="docs-main">
+        <aside class="sidebar">
+            <nav class="sidebar-nav">
+                <ul class="sidebar-nav-links">
+                    <li>
+                        <a class="sidebar-nav-link" href="/Avatar/avatar.html">Avatar</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Alert/alert.html">Alert</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Badges/badges.html">Badges</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Button/button.html">Buttons</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Card/card.html">Cards</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Image/image.html">Image</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Input/input.html">Input</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Text-utilities/text-utilities.html">Text</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link" href="/Rating/rating.html">Rating</a>
+                    </li>
+                    <li>
+                        <a class="sidebar-nav-link sidebar-link-active" href="/Snackbar/snackbar.html">Snackbar</a>
+                    </li>
+                </ul>
+            </nav>
+        </aside>
+        <section class="docs-main-content">
+            <h1 class="snackbar-heading">Snackbar</h1>
+            <p class="snackbar-info">Snackbars are often used as a tooltips/popups to show a message at the bottom of
+                the screen.</p>
+
+            <div class="snackbar">
+                <div class="snackbar-text">
+                    Can't send photo. Retry in 5 seconds.
+                </div>
+
+                <div class="snackbar-button">
+                    RETRY
+                </div>
+
+                <div class="snackbar-cross-icon">
```

Comment:
Using cross icon instead of putting X would be better, in my opinion

Reply:
Will work on that. Thank you.